modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoGinServerCodegen.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
@kemokemo I believe we no longer need line 3 for new files added directly to this project (OpenAPI Generator). Please refer to https://opensource.stackexchange.com/a/7301/12997 for more information.

Reply:
Thank you for your review and the information about the 'Copyright notice in the file header'.
I'll delete these 3 lines.